OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -32,7 +32,7 @@ public enum UncloakType
 		Damage = 64,
 		Heal = 128,
 		SelfHeal = 256,
-		Dock = 512,
+		Link = 512,
```

Comment:
Should this be split into `LinkHost` and `LinkClient`?

Reply:
I'm not sure

Reply:
It would only matter if an actor would both be able to be linked to and to link itself